recipes-kernel/linux/linux-raspberrypi.inc

```diff
@@ -21,7 +21,7 @@ KERNEL_DEFCONFIG_raspberrypi2 ?= "bcm2709_defconfig"
 KERNEL_DEFCONFIG_raspberrypi3-64 ?= "bcmrpi3_defconfig"
```

Comment:
Can you please modify the commit log in accordance to http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines ? Looks good to me other than that.